portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -35,16 +35,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     PrintWriter out = response.getWriter();
     UserService userService = UserServiceFactory.getUserService();
     String logoutUrl = userService.createLogoutURL("/index.html");
-    if (userService.isUserLoggedIn()){
-        Login login = new Login(true, "", logoutUrl);
-        out.println(new Gson().toJson(login));
-    }
-    else{
-        String loginUrl = userService.createLoginURL("/index.html");
-        Login login = new Login(false, loginUrl, logoutUrl);
-        out.println(new Gson().toJson(login));
-    }
+    String loginUrl = userService.createLoginURL("/index.html");
+    Login login = userService.isUserLoggedIn() ? 
+    new Login(true, "", logoutUrl) : new Login(false, loginUrl, logoutUrl);
+    out.println(new Gson().toJson(login));
   }
 }
```

Comment:
nit: trailing new line

Reply:
there's a new line in my actual file - not sure why it's not showing up here in github